src/test_utils/test-utils.tsx

```diff
@@ -1,5 +1,10 @@
 import React from "react";
-import { render, queries, RenderResult } from "@testing-library/react";
```

Comment:
Did some typescript updates in this file so the exported functions are properly typed.